docs/source/_static/juju/bundle.yaml

```diff
@@ -0,0 +1,155 @@
+envExport:
+  services:
+    cinder:
+      charm: "cs:trusty/cinder-136"
+      num_units: 1
+      annotations:
+        "gui-x": "1045.973572253264"
+        "gui-y": "544.8983810601128"
+      to:
+        - "calico-acl-manager=0"
+    "neutron-calico":
+      charm: "cs:~cory-benfield/trusty/neutron-calico-7"
```

Comment:
We can do, but it's going to be a fair bit of trouble as the merge requests and charm store requests are all linked to my personal space. Might be easier just to add the calico team to the branches?